smoreo:
  camera_height_from_ground: 0.45
  cone_height: 0.38
  cut_off_y: 1000
  cx: 168.474960327
  cy: 97.19174194
  f: 167.377334594726
